java/src/Ice/src/main/java/com/zeroc/Ice/SyscallException.java

```diff
@@ -1,54 +1,28 @@
-//
-// Copyright (c) ZeroC, Inc. All rights reserved.
-//
-//
-// Ice version 3.7.10
-//
-// <auto-generated>
-//
-// Generated from file `LocalException.ice'
-//
-// Warning: do not edit this file.
-//
-// </auto-generated>
-//
+// Copyright (c) ZeroC, Inc.
 
 package com.zeroc.Ice;
 
-/**
- * This exception is raised if a system error occurred in the server or client process. There are
- * many possible causes for such a system exception. For details on the cause, {@link
- * SyscallException#error} should be inspected.
- */
+/** This exception is raised if a system error occurred in the server or client process. */
 public class SyscallException extends LocalException {
   public SyscallException() {
-    this.error = 0;
+    super();
   }
 
   public SyscallException(Throwable cause) {
```

Comment:
I would add a message here when calling super.

Reply:
After talking about it, it's fine to keep this with a `null` message,
since calling `toString()` on an exception will still return a useful string, even without a user-supplied message.